### DISCUSSION TITLE
DieMobs doesn't work

### DISCUSSION THREAD

**Discussion by heelsqvtjj** [Pryaxis/TShock#2212] [Q&A] [answered]
Hello!

I updated my server for the new version but I can't get "DieMobs" to work. I tried so much but everything doesn't seems to work... does anybody has an idea? 

Thanks c:

![Capture](https://user-images.githubusercontent.com/78381085/106509282-b6a0d900-64cd-11eb-80b1-b7fb05e20755.PNG)

**Comment by Kojirremer** [comment, marked as answer]
https://github.com/newyrose/DieMob/tree/master/bin/Release
Try this version. Yours is most likely broken.

**Reply by heelsqvtjj** [reply]
Thanks! Its working now c:
